relay-event-schema/src/protocol/event.rs

```diff
@@ -480,6 +480,11 @@ pub struct Event {
     #[metastructure(omit_from_schema)] // we only document error events for now
     pub breakdowns: Annotated<Breakdowns>,
 
+    /// Information about attempts to scrape a JS source or sourcemap file from the web.
+    /// This field is populated by sentry.
+    #[metastructure(omit_from_schema)] // not part of external schema
+    pub scraping_attempts: Annotated<Object<Value>>,
```

Comment:
Does `Object` here mean something like a JSON object? The field is actually an array.

Reply:
Good catch, it's an alias for a `BTreeMap`. I will change it and add a test case.